sagemaker_batch_transform/pytorch_flores_batch_transform/sagemaker_batch_transform_torchserve.ipynb

```diff
@@ -0,0 +1,377 @@
+{
```

Comment:
<p>Recommended to add Cleanup and Conclusion sections.</p><p> </p><p>Cleanup - delete any resources created in the notebook.</p><p> </p><p>Conclusion - what did the customer learn? How can they apply or adapt this going forward?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
@nskool @jkroll-aws for Conclusion something along this may work?, " This notebook showed the steps to set up a Sagemaker batch trasnsform job that uses Torchserved under the hood for serving the model, this is useful to test production variants, different models or hyperparamters using a test dataset. To adopt this work to other applications, users can write their own custom handlers for Torchserve that decides the model initialization, data pre and post processing and inference logic along with other consideration about setting the batch transform job discussed in this blog post[link to blog post]."

Reply:
Done, added this in updated PR.